invoke.example.yml

```diff
@@ -1,9 +1,9 @@
 ---
 nautobot_firewall_models:
-  project_name: "nautobot-firewall-model"
-  nautobot_ver: "1.1.0"
+  project_name: "nautobot_firewall_models"
+  nautobot_ver: "1.3.5"
```

Comment:
Good catch but doesn't factor in I believe, should be removed. Similar to issue 219 for internal standards

Reply:
@whitej6 to take care of this.